source/gui/sdl/video.c

```diff
@@ -113,7 +113,9 @@ gui_sdl_video_init(
     SDL_GL_SetSwapInterval(app->video.vsync);
 
     /* Initialize OpenGL */
-    if (glewInit()) {
+    int err = glewInit();
```

Comment:
```suggestion
    err = glewInit();
```

Would you mind declaring `int err` at the beginning of the function instead? :D